### PR TITLE
Fixing compatibility issues with Python3

### PR DESCRIPTION
**Pull request by Ivanca**
**Background:** An effort is being made to move everything from Python 2.7 to 3, including making this repository compatible with both as specified on this JIRA issue: https://openedx.atlassian.net/projects/INCR/issues/INCR-18?filter=allopenissues and these changes aim to do that.

**Studio Updates:** None.

**LMS Updates:** None

Testing: Same ones as before (and have been updated to be compatible with both as well), passing.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Ivanca! I've created [OSPR-2511](https://openedx.atlassian.net/browse/OSPR-2511) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx/django-config-models/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by Ivanca** [comment]
I don't fully understand what the "codevob" is asking for here; more clarity in the changes that need to be made (if any) would be appreciated.

File: config_models/models.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from rest_framework.utils import model_meta
-
+import six
```
Comment (jmbowman) [inline config_models/models.py]: I don't see `six` among the dependencies in `setup.py` yet.  Maybe just use the version bundled with `django.utils` as you used in the file below?

Comment (Ivanca) [reply config_models/models.py]: Yeah; fixed now on last commit (imports from django.utils)

**Comment by jmbowman** [comment]
The errors under Python 3.6 are due to the `futures` dependency in `requirements/test.txt`; it's a backport of native Python 3 functionality and hence hasn't been packaged for installation in a Python 3 environment.  You should be able to fix that by adding `futures ; python_version == "2.7"` to `requirements/test.in` (a [similar fix](https://github.com/edx/bok-choy/blob/master/requirements/test.in#L6) was made in bok-choy).  The need to work around this manually is a [bug in pip-tools](https://github.com/jazzband/pip-tools/issues/563).

**Comment by Ivanca** [comment]
Finally all checks passing now  👍
Travis CI is testing on both 2.7 and 3.6

File: setup.py [hunk 2]
```diff
@@ -62,5 +62,6 @@ def get_version(*file_paths):
         'Natural Language :: English',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```
Comment (jmbowman) [inline setup.py]: Should probably add an entry for 3.6 here also.

**Comment by Ivanca** [comment]
done

**Comment by edx-webhook** [comment]
@Ivanca 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/django-config-models&entry.867055334=https://github.com/edx/django-config-models/pull/17&entry.1484655318=https://github.com/Ivanca&entry.752974735=2018-07-09+19:30&entry.1917517419=2018-07-12+19:59&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**